package.json

```diff
@@ -54,9 +54,14 @@
       "@unhead/vue",
       "@unhead/schema",
       "#nuxt-scripts",
-      "#nuxt-scripts-validator"
+      "#nuxt-scripts-validator",
+      "third-party-capital",
+      "knitwork"
     ]
   },
+  "peerDependencies": {
+    "third-party-capital": "^1.0.30"
```

Comment:
what is the rationale for this?

Reply:
I moved it to peer dep so module managers won't enforce a specific version of tpc. User will be free to update tpc on their own. 

I'll change it to >=1

Reply:
What's the issue with having it as a dependency though? It's not clear

Reply:
Having it as a dependency prevent users from having for example tpc v1.2.30. If set in peerDeps tpc should be updatable by users without tpc's version being locked by @nuxt/scripts

Reply:
Seems like a reasonable constraint to avoid breaking behaviours. If TPC introduces new scripts or config we'd need to add code to support them anyway.